logger/zap.go

```diff
@@ -0,0 +1,168 @@
+package logger
+
+import (
+	"log"
+	"os"
+
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+)
+
+type ZapLogger struct {
+	Zap *zap.SugaredLogger
+}
+
+var zapLogger *ZapLogger
+
+func init() {
+	logLevelEnv := os.Getenv("LOG_LEVEL")
+	var logLevel zapcore.Level
+	switch logLevelEnv {
+	case "DEBUG":
+		logLevel = zapcore.DebugLevel
+	case "INFO":
+		logLevel = zapcore.InfoLevel
+	case "WARN":
+		logLevel = zapcore.WarnLevel
+	case "ERROR":
+		logLevel = zapcore.ErrorLevel
+	case "DPANIC":
+		logLevel = zapcore.DPanicLevel
+	case "PANIC":
+		logLevel = zapcore.PanicLevel
+	case "FATAL":
+		logLevel = zapcore.FatalLevel
+	default:
+		logLevel = zapcore.DebugLevel
+	}
+	level := zap.NewAtomicLevel()
+	level.SetLevel(logLevel)
+	cfg := zap.Config{
+		Level:    level,
+		Encoding: "json",
+		EncoderConfig: zapcore.EncoderConfig{
+			TimeKey:        "TimeStamp",
+			LevelKey:       "Severity",
+			NameKey:        "Name",
+			CallerKey:      "Caller",
+			MessageKey:     "Body",
+			EncodeLevel:    zapcore.CapitalLevelEncoder,
+			EncodeTime:     zapcore.ISO8601TimeEncoder,
+			EncodeDuration: zapcore.StringDurationEncoder,
+			EncodeCaller:   zapcore.ShortCallerEncoder,
+		},
+		OutputPaths: []string{"stdout", "/tmp/zap.log"},
```

Comment:
The target of logging output must be configurable by the global config file.

Reply:
use config file
https://github.com/hyperledger-labs/yui-relayer/pull/92/commits/5975bdcd218e278686807526ad9b6526ffb3ade7